tests/utils/utils.go

```diff
@@ -327,6 +328,17 @@ func CreateReceiveCrossChainMessageTransaction(
 
 	gasFeeCap, gasTipCap, nonce := CalculateTxParams(ctx, rpcClient, fundedAddress)
 
+	if alterMessage {
+		// Alter the message
+		modifiedMessage := make([]byte, len(signedMessage.Payload))
+		copy(modifiedMessage, signedMessage.Payload)
+		modifiedMessage[0] = ^modifiedMessage[0]
+		Expect(modifiedMessage[:]).ShouldNot(Equal(signedMessage.Payload[:]))
+		signedMessage.Payload = modifiedMessage
+
+		signedMessage.Initialize()
```

Comment:
Should we consider a related test where the signature is from the original payload? Perhaps testing incorrect signatures is already covered in a different test case.